pennylane/devices/default_qubit.py

```diff
@@ -360,6 +359,15 @@ def _get_unitary_matrix(self, unitary):  # pylint: disable=no-self-use
 
         return unitary.matrix
 
+    @classmethod
+    def capabilities(cls):
+        capabilities = super().capabilities().copy()
+        capabilities.update(
+            supports_reversible_diff=True,
```

Comment:
How come this would not be defined among the capabilities in `Device`?

Reply:
It is set to False in ``Device``...

Reply:
Oh not sure how I missed that, my bad!